packages/dbs-ai/client/views/app/tabbar/smarti.js

```diff
@@ -0,0 +1,86 @@
+Template.dbsAI_smarti.onCreated(function() {
+	this.helpRequest = new ReactiveVar(null);
+
+	const instance = this;
+
+	this.autorun(() => {
+		if (instance.data.rid) {
+
+			if (!instance.helpRequest.get()) { //todo remove after PoC: Non-reactive method call
+				Meteor.call('assistify:helpRequestByRoomId', instance.data.rid, (err, result) => {
+					if (!err) {
+						instance.helpRequest.set(result);
+					} else {
+						console.log(err);
+					}
+				});
+			}
+		}
+	});
+
+});
+
+Template.dbsAI_smarti.onDestroyed(function() {
+	clearTimeout(this.loading);
+});
+
+/**
+ * Create Smarti (as soon as the script is loaded)
+ */
+Template.dbsAI_smarti.onRendered(function() {
+
+	var self = this;
+
+	function createSmarti() {
+		if(window.SmartiWidget == undefined) {
+			console.log('Smarti is undefined');
+			self.loading = setTimeout(createSmarti,200);
+		} else {
+
+			var DBS_AI_Redlink_URL =
+				RocketChat.settings.get('DBS_AI_Redlink_URL').endsWith('/') ?
+					RocketChat.settings.get('DBS_AI_Redlink_URL') :
+				RocketChat.settings.get('DBS_AI_Redlink_URL') + '/';
+
+			var WEBSOCKET_URL =
+				"ws" + RocketChat.settings.get('Site_Url').substring(4) + "websocket/";
+
+
+			self.smarti = new window.SmartiWidget(self.find('.external-message'), {
+				socketEndpoint: WEBSOCKET_URL,
+				smartiEndpoint: DBS_AI_Redlink_URL,
+				channel: self.data.rid,
+				inputCssSelector:'.autogrow-shadow'
```

Comment:
@tkurz This is about the input box (in order to copy to), right? if so, what about `$('#chat-window-${ instance.data.roomId } .input-message')`